pycheribuild/projects/cross/cheribsd.py

```diff
@@ -1759,7 +1766,7 @@ class BuildCheriBsdMfsKernel(BuildCHERIBSD):
     target = "cheribsd-mfs-root-kernel"
     dependencies = ["disk-image-mfs-root"]
     repository = ReuseOtherProjectRepository(source_project=BuildCHERIBSD, do_update=True)
-    supported_architectures = CompilationTargets.ALL_CHERIBSD_RISCV_TARGETS
+    supported_architectures = CompilationTargets.ALL_CHERIBSD_TARGETS_WITH_HYBRID
```

Comment:
This includes amd64 which doesn't have an MFS root config committed

Reply:
Thanks.  Is `ALL_CHERIBSD_RISCV_TARGETS + ALL_CHERIBSD_MORELLO_TARGETS` the right thing to use?

Reply:
That doesn't include plain AArch64 :)